util/emailer.py

```diff
@@ -33,7 +34,8 @@ def send(self):
 
         server = smtplib.SMTP(Email.server, Email.port)
         server.starttls()
-        server.login(Email.address, Email.password)
+        password = base64.b64decode(Email.password).decode('utf-8')
```

Comment:
I'm wondering if we shouldn't decrypt the password when we fetch it from the config over in configuration.py so that we keep imports to a minimum and so that anyone else working on this project in the future doesn't have to remember to do it themselves when they need a password.

Typing that made me realize that maybe we should add a method or two to the extensions for encoding and decoding strings as I'm sure this will be useful in other projects (and maybe even at some point in the future in this project).

Reply:
What do you think?

Reply:
Thats a fair point. Thanks for pointing that out.

Reply:
So, as a note, the way the current code for Emailer is an issue. When testing the emailing feature, you will have to update line 38 from `server.sendmail(Email.address, self.recipients, msg.as_string())` to `server.sendmail(Email.address, Email.recipients, msg.as_string())` or you'll get an error. There may be a few other changes needed that I'm not remembering.